changelogs/unreleased/gh-6789-fix-memleak-in-vdbe.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/sql
```

Comment:
I had an impression that we are supposed to put NO_* tags in the end of the commit message (but before docbot request). So here it would be after `Closes`. But I don't know if there is a standard, so up to you. Besides, `NO_TEST` is not checked from what I see. So you don't need to write it.

Reply:
I used this as an example: https://github.com/tarantool/tarantool/pull/6856

However, I agree that placing these tags after `Closes` makes it better.

Reply:
Which tags are parsed right now you can check in `tools/check-commits` script. Maybe there is also a doc somewhere, but I didn't find.

Reply:
I added `NO_TEST` in PR #6856 because it looks as a good companion of `NO_CHANGELOG` and `NO_DOC`: it is quite intuitive to find this information together. My motivation was not based on whether it is validated or not.

However @locker said that he want to add such check in a future.